cmd/share-secret/storedata.go

```diff
@@ -0,0 +1,27 @@
+package main
+
+import (
+	"time"
+
+	"github.com/google/uuid"
+)
+
+type StoreData struct {
```

Comment:
Although this probably works, there is a lot of data here that really has no use. I would remove everything but `Secret` field